### DISCUSSION TITLE
How to use and Implement TripletMarginwithDistance?

### DISCUSSION THREAD

**Discussion by vazand** [mrdbourke/pytorch-deep-learning#291] [General]
Hi Guys,
        I'm a beginner to PyTorch, However, using learnpytorch.io I've learned how to train a model and etc.., I've also learned how to use CNN to train and predict an object. but for some reason, I have to create a face recognition model, I've tried to understand TripletMarginwithDistance and I don't know how to use this loss function in my problem. can anyone explain how to use this loss function in a face recognition problem?
